tests/test_sensor.py

```
@@ -0,0 +1,93 @@
+    MyAirSleepRecordSensor,
+    SensorEntityDescription,
+)
+
+
+class DummyCoordinator:
+    def __init__(self, data):
+        self.data = data
+
+
+@pytest.mark.parametrize(
+    "data,expected",
```

Ruff (0.12.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)